firestore.rules

```diff
@@ -32,6 +32,10 @@ service cloud.firestore {
       allow read;
     }
 
+    match /{somePath=**}/portfolioHistory/{portfolioHistoryId} {
```

Comment:
I actually added this just above! I had thought it was necessary for the last change (b/c saw it in the dev rules), although now I see that it wasn't